services/brig/.env

```diff
@@ -4,3 +4,7 @@ BRIG_CASSANDRA_PORT=9042
 BRIG_CASSANDRA_KEYSPACE=brig_test
 BRIG_ELASTICSEARCH_URL=http://localhost:9200
 BRIG_ELASTICSEARCH_USER_INDEX=directory_test
+# AWS_SQS_ENDPOINT=http://localhost:4568
```

Comment:
Unfortunately these `.env` files don't support any comments (at least when loaded from keiretsu). Perhaps `localhost` should be the default in this `.env` files, i.e. the default when testing.

Reply:
👍